lib/Conch/Controller/User.pm

```diff
@@ -134,16 +149,19 @@ Get the individual key/value pair for a setting for the User
 sub get_setting ($c) {
 	my $key = $c->param('key');
 
-	my $user = Conch::Model::User->lookup( $c->stash('user_id') );
+	my $user = $c->stash('user');
 	Mojo::Exception->throw('Could not find previously stashed user')
 		unless $user;
 
-	my $settings = $user->settings;
+	my $setting = $user->user_settings->active->search(
+		{ name => $key },
+		{ order_by => { -desc => 'created' } },
+	)->first;
 
 	return $c->status( 404, { error => "No such setting '$key'" } )
-		unless $settings->{$key};
+		unless $setting;
 
-	$c->status( 200, { $key => $settings->{$key} } );
+	$c->status( 200, { $key => from_json($setting->value) } );
```

Comment:
the 'status' helper should be handling the serialization for you at https://github.com/joyent/conch/blob/master/lib/Conch.pm#L109

*should* is operative, I think.

Reply:
this is deserialization, from the db.

Reply:
Roger